packages/aws-rfdk/lib/deadline/lib/thinkbox-docker-recipes.ts

```diff
@@ -106,6 +106,8 @@ export class ThinkboxDockerRecipes extends Construct {
 
     this.version  = props.stage.getVersion(this, 'Version');
 
+    // this.versionQuery  = props.stage.getVersion(this, 'Version');
```

Comment:
Looks like this was here for testing, and can be removed.

Reply:
Removed